data/wicked/xml/ipip.xml

```diff
@@ -13,3 +13,12 @@
     </address>
   </ipv4:static>
 </interface>
+
+<interface>
+  <name>iface</name>
+  <ipv4:static>
+    <address>
+      <local>local_ip/15</local>
```

Comment:
why not 24 ?

Reply:
good question, I was just following same approach we did in the past..

Reply:
at least when in same config we have another interface configured with 24 submask I think it make sense to do the same for another